src/components/button/styled.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import styled, { css } from "styled-components"
+import { Button } from "@rmwc/button"
+import { getColor } from "../../theme/utils"
+
+export const StyledButton = styled(({ ...otherProps }) => <Button {...otherProps} />)`
```

Comment:
Have you tried just styled(Button)? I have a feeling they improved that.

Reply:
But I have conditional CSS bellow based on the props. Would it work as you say? (it doesn't seem so from a fast test I did)